catalyst/callbacks/sklearn_classifier.py

```diff
@@ -0,0 +1,179 @@
+            )
+            sampler = data.BalanceBatchSampler(labels=train_dataset.get_labels(), p=5, k=10)
+            train_loader = DataLoader(
+                dataset=train_dataset,
+                sampler=sampler,
+                batch_size=sampler.batch_size)
+
+            valid_dataset = datasets.MNIST(root=os.getcwd(), transform=transforms, train=False)
+            valid_loader = DataLoader(dataset=valid_dataset, batch_size=1024)
+
+            # 2. model and optimizer
+            model = models.MnistSimpleNet(out_features=16)
+            optimizer = Adam(model.parameters(), lr=0.001)
+
+            # 3. criterion with triplets sampling
+            sampler_inbatch = data.HardTripletsSampler(norm_required=False)
+            criterion = nn.TripletMarginLossWithSampler(
+                margin=0.5,
+                sampler_inbatch=sampler_inbatch
+            )
+
+            # 4. training with catalyst Runner
+            class CustomRunner(dl.SupervisedRunner):
+                def handle_batch(self, batch) -> None:
+                    images, targets = batch["features"].float(), batch["targets"].long()
+                    features = self.model(images)
+                    self.batch = {
+                        "embeddings": features,
+                        "targets": targets,
+                    }
+
+            callbacks = [
+                dl.ControlFlowCallback(
+                    dl.CriterionCallback(
+                        input_key="embeddings",
+                        target_key="targets",
+                        metric_key="loss"),
+                    loaders="train",
+                ),
+                dl.SklearnClassifierCallback(
+                    feature_key="embeddings",
+                    target_key="targets",
+                    train_loader="train",
+                    valid_loader="valid",
+                    sklearn_classifier_fn=LogisticRegression,
+                    sklearn_metric_fn=partial(top_k_accuracy_score, **{"k": 1}),
+                ),
+            ]
+
+            runner = CustomRunner(input_key="features", output_key="embeddings")
+            runner.train(
+                model=model,
+                criterion=criterion,
+                optimizer=optimizer,
+                callbacks=callbacks,
+                loaders={"train": train_loader, "valid": valid_loader},
+                verbose=False,
+                logdir="./logs",
+                valid_loader="valid",
+                valid_metric="sklearn_classifier_metric",
+                minimize_valid_metric=False,
+                num_epochs=100,
+            )
+    """
+
+    def __init__(
+        self,
+        feature_key: str,
+        target_key: str,
+        train_loader: str,
+        valid_loader: str,
+        sklearn_classifier_fn: Callable,
+        sklearn_metric_fn: Callable,
+    ) -> None:
+        super().__init__()
+        self.order = CallbackOrder.Internal

```

Comment:
could you please pass order into `super().__init__()`? It should be natively supported for and `Callback` child

Reply:
Sure. But what do you think about CallbackOrder.Metric here?

Reply:
**IF** we have a `metric_fn` here - yes, **IF-NOT** - `Internal` should be a good choice

Reply:
I understood, thanks.